src/ert/_c_wrappers/job_queue/workflow_job.py

```diff
@@ -44,7 +61,7 @@ class WorkflowJob:
     internal: bool
     min_args: Optional[int]
     max_args: Optional[int]
-    arg_types: List[ContentTypeEnum]
+    arg_types: List[SchemaItemType]
```

Comment:
Won't this still become `ContentTypeEnum` when you use the old parser?

Reply:
Fixed